noipy/utils.py

```diff
@@ -21,10 +19,10 @@ def read_input(message):
     return input(message)
 
 
-def get_ip():
+def get_ip(httpbin_url):
```

Comment:
Since this method is being used in [other parts of the code](https://github.com/pv8/noipy/blob/b4bf6557f46d138f8f4965cd85935889cb6fdc72/noipy/main.py#L110) (test coverage improvement needed 😞), what about attributing a default value to the new function parameter?

```suggestion
def get_ip(req_ip_url=HTTPBIN_URL):
```